interfaces/orientation-sensor.idl

```diff
@@ -0,0 +1,15 @@
+typedef (Float32Array or Float64Array or DOMMatrix) RotationMatrixType;
+
+[SecureContext, Exposed=Window]
+interface OrientationSensor : Sensor {
```

Comment:
Does idlharness check that constructor is not defined to verify that interface is not constructible?

Reply:
Seems the answer is no, it is in TODO list at: https://github.com/w3c/web-platform-tests/blob/master/resources/idlharness.js#L1155